Changes

```diff
@@ -1,5 +1,38 @@
 Release history of Zonemaster
 
+v2020.1 2020-11-06 (public release version)
+
+ [Features]
+ - Added support for Norwegian language, see Zonemaster-Engine,
+   zonemaster-cli and zonemaster-backend (not yet supported in
+   zonemaster-gui).
+ - Update implementation of Connectivity03 with added support
+   of RIPE Ris whois for ASN lookup, see Zonemaster-Engine.
+ - Test case specifications
+   - Updated Connectivity03 (#892, #893)
+   - Create Basic04 (#879, #859)
+
+ [Fixes]
+ - Test case specifications
+   - Fixed Nameserver02 (#891)
+   - Fixed Connectivity02 (#888, #646)
+   - Fixed Nameserver12 (#878)
+   - Fixed Nameserver13 (#880)
```

Comment:
Maybe we should say something about what was fixed?

Reply:
@mattias-p, please see #907 and review that.